packages/components/src/internal/components/domainproperties/dataset/DatasetPropertiesAdvancedSettings.tsx

```diff
@@ -140,8 +140,6 @@ export class AdvancedSettings extends React.PureComponent<AdvancedSettingsProps,
         super(props);
         const initialState = this.getInitialState();
 
-        initQueryGridState(); // needed for selectRows usage
```

Comment:
Thanks for cleaning this up. You can also remove the import statement. Same in SampleTypeDesigner.tsx.